modules/r2dbc/src/main/java/org/testcontainers/r2dbc/R2dbcMSSQLServerContainer.java

```diff
@@ -0,0 +1,22 @@
+package org.testcontainers.r2dbc;
+
+import io.r2dbc.mssql.MssqlConnectionConfiguration;
+import io.r2dbc.mssql.MssqlConnectionFactory;
+import io.r2dbc.spi.ConnectionFactory;
+import org.testcontainers.containers.MSSQLServerContainer;
+
+/**
+ * @author humblehound
```

Comment:
Although we don't have a clear policy for `@author` tags in the project, I personally prefer to omit them in regards to shared code ownership.

Reply:
I don't feel strong about them either, just followed what I already found in the repo. Will remove